bower.json

```diff
@@ -32,6 +32,7 @@
     "leaflet-tilelayer-geojson": "*",
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "Leaflet.awesome-markers": "*",
+    "Leaflet.vector-markers": "leaflet.vector-markers#0.0.3",
```

Comment:
Ah, the library is registered at Bower with all letters written in lowercase. Thus, Bower added the repository name in the value part. I'll fix that.